runner/dl_checkpoints.sh

```diff
@@ -79,7 +79,42 @@ function download_all_models() {
 
     # Download live-video-to-video models.
     huggingface-cli download KBlueLeaf/kohaku-v2.1 --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
-    huggingface-cli download KwaiVGI/LivePortrait --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download stabilityai/sd-turbo --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download warmshao/FasterLivePortrait --local-dir models/FasterLivePortrait--checkpoints
+}
+
+function build_tensorrt_models() {
+    download_all_models
+
+    printf "\nBuilding TensorRT models...\n"
+
+    # Matrix of models and timesteps to compile StreamDiffusion TensorRT engines for.
+    MODELS="stabilityai/sd-turbo KBlueLeaf/kohaku-v2.1"
+    TIMESTEPS="3 4" # This is basically the supported sizes for the t_index_list
+    docker run --rm -it -v ./models:/models --gpus all \
+        livepeer/ai-runner:live-app-streamdiffusion \
```

Comment:
```suggestion
        livepeer/ai-runner:live-app-streamdiffusion \
        -e PIPELINE=live-video-to-video -e MODEL_ID=streamdiffusion \
```
I had to include these two env variables for the container to stay running while tensorrt is compiled

Reply:
I actually don't think those envs would be necessary since the scripts that we run in here don't check those variables (they are very simple `tensorrt` compilation scripts). Did you notice that I changed the command that I run within the container when starting them here? Were you doing that as well when you ran these?

If you were running exactly this script and still got the errors, can you share the terminal output here?